tests/integration/cloudformation/resources/test_ssm.py

```diff
@@ -119,3 +119,31 @@ def test_update_ssm_parameter_tag(deploy_cfn_template, aws_client):
     #
     # ssm_tags = aws_client.ssm.list_tags_for_resource(ResourceType="Parameter", ResourceId=parameter_name)['TagList']
     # assert ssm_tags == []
+
+
+def test_deploy_patch_baseline(deploy_cfn_template, aws_client, snapshot):
```

Comment:
the tests here should be validated and annotated with `pytest.mark.aws_validated` :thinking: 

any particular reason this was skipped here?